static_test_driver/static_test_driver.ino

```diff
@@ -169,44 +159,33 @@ void setup() {
 
 void loop() {
   // Grab force data
-  #if CONFIGURATION == MK_2
   // TODO: This hangs when the load cell amp isn't connected. Figure out a way to check this first.
   force = scale.get_units(); // Force is measured in lbs
-
-  #else
-  float count = analogRead(FORCE_PLATE_PIN);
-  float voltage = count / 1023 * 5.0;// convert from count to raw voltage
-  force = FORCE_PLATE_INTERCEPT + voltage * FORCE_PLATE_SLOPE - force_zero_val; //converts voltage to sensor reading
-
-  force_hist_vals[force_val_num] = force;
-  force_val_num++;
-  if (force_val_num >= FORCE_PLATE_NUM_HIST_VALS) {
-    force_val_num = 0;
-    force_zero_ready = true;
-  }
-  #endif
-  // TODO: Error checking
+  
+  // TODO: Error checking for load cell?
   
   // Grab thermocouple data for Mk.2
   #if CONFIGURATION == MK_2
-  chamber_temp = read_thermocouple("Chamber", THERMO3_LED, chamber_thermocouple, chamber_temp_error);
+  chamber_temp[0] = read_thermocouple("Chamber 1", THERMO1_LED, chamber_thermocouple_1, chamber_temp_error[0]);
+  chamber_temp[1] = read_thermocouple("Chamber 2", THERMO2_LED, chamber_thermocouple_2, chamber_temp_error[1]);
+  chamber_temp[2] = read_thermocouple("Chamber 3", THERMO3_LED, chamber_thermocouple_3, chamber_temp_error[2]);
+  #endif
+  
   
   // Grab pressure data
-  pressure = (analogRead(PRESSURE_PIN) * 5 / 1024.0) * PRESSURE_CALIBRATION_FACTOR - (PRESSURE_OFFSET + pressure_zero_val); // Pressure is measured in PSIG
+  pressure_fuel = (analogRead(PRESSURE_FUEL) * 5 / 1024.0) * PRESSURE_CALIBRATION_FACTOR - (PRESSURE_OFFSET + pressure_zero_val_1); // Pressure is measured in PSIG
+  pressure_ox = (analogRead(PRESSURE_OX) * 5 / 1024.0) * PRESSURE_CALIBRATION_FACTOR - (PRESSURE_OFFSET + pressure_zero_val_2); // Pressure is measured in PSIG
   // TODO: Error checking
 
   // Update pressure tare data
-  pressure_hist_vals[pressure_val_num] = pressure;
-  pressure_val_num++;
-  if (pressure_val_num >= PRESSURE_NUM_HIST_VALS) {
-    pressure_val_num = 0;
-    pressure_zero_ready = true;
-  }
-  #endif
+  tare_pressure (PRESSURE_FUEL);
```

Comment:
I don't see this method defined anywhere - is this in another file that you forgot to commit?

Reply:
Oh nvm, I see you opened a seperate pull request.